otherlibs/stdune/test/path_tests.ml

```diff
@@ -460,3 +460,37 @@ let%expect_test _ =
 None
 |}]
 ;;
+
+let%expect_test "drop prefix" =
+  Path.drop_prefix ~prefix:(r "foo/bar") (r "foo/bar/baz")
+  |> Dyn.option Path.Local.to_dyn
+  |> print_dyn;
+  [%expect {| Some "baz" |}]
+;;
+
+let%expect_test "drop external prefix" =
+  Path.External.drop_prefix
+    ~prefix:(e "foo/bar" |> Path.as_external_exn)
```

Comment:
How about just creating the external paths directly instead of relying on converting them with this function?

Reply:
Done.